content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -274,9 +274,6 @@ different Kubernetes components.
 | `VolumeScheduling` | `true` | Beta | 1.10 | 1.12 |
 | `VolumeScheduling` | `true` | GA | 1.13 | - |
 | `VolumeSubpath` | `true` | GA | 1.13 | - |
-| `VolumeSubpathEnvExpansion` | `false` | Alpha | 1.14 | 1.14 |
-| `VolumeSubpathEnvExpansion` | `true` | Beta | 1.15 | 1.16 |
-| `VolumeSubpathEnvExpansion` | `true` | GA | 1.17 | - |
```

Comment:
This removal is incorrect. Graduated feature gates remain in the list of feature gates, in the section _Feature gates for graduated or deprecated features_